static/app/views/dashboardsV2/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -77,7 +86,7 @@ export function ReleaseSearchBar({orgSlug, query, projectIds, onSearch, onBlur}:
           maxQueryLength={MAX_QUERY_LENGTH}
           maxSearchItems={MAX_SEARCH_ITEMS}
           searchSource="widget_builder"
-          query={query.conditions}
+          query={widgetQuery.conditions}
```

Comment:
We only seem to use `widgetQuery` to get `conditions` from it. What do you think about just passing down `conditions`?

Reply:
eh I think it's fine to pass the widget query for now, it's a nice pattern that matches the rest of the config. we can clean it up in a follow up if necessary